provider/tests/test_mgmt_cmd.py

```diff
@@ -0,0 +1,58 @@
+"""
+Unittests for testing management commands.
+"""
+import unittest
+import mock
+from django.core.management import CommandError, call_command
+from provider.management.commands.delete_expired_grant_tokens import Command
+
+
+class TestDeleteExpiredGrantTokens(unittest.TestCase):
+    """
+    Tests for parsing arguments for the `delete_expired_grant_tokens` management command.
+    """
+    def setUp(self):
+        super(TestDeleteExpiredGrantTokens, self).setUp()
+        self.command = Command()
+        self.command_name = 'delete_expired_grant_tokens'
+
+    @mock.patch('provider.oauth2.models.Grant.objects')
+    def test_no_args(self, objects):
+        objects.raw = mock.Mock(return_value=10)
+        self.command.handle(self.command_name)
+        objects.raw.assert_has_calls(
```

Comment:
Leave it, I don't think it's worth the effort. We can verify in the DB after we run this.